lib/resources/BitcoinReceivers.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+var StripeResource = require('../StripeResource');
+var stripeMethod = StripeResource.method;
+
+module.exports = StripeResource.extend({
+
+  path: 'bitcoin/receivers',
+
+  includeBasic: [
+    'create', 'list', 'retrieve'
+  ],
+
+  listBitcoinTransactions: stripeMethod({
+    method: 'GET',
+    path: '/{receiverId}/transactions',
```

Comment:
they're immutable from the user API, so not that I can think of. These transactions represent the transfer of bitcoins to fill the receiver before it's used in a charge -- they're very specific to the receiver itself.